lib/winston-loggly.js

```diff
@@ -139,9 +155,9 @@ Loggly.prototype.log = function (level, msg, meta, callback) {
 // #### @options {Object} Set stream options
 // Returns a log stream.
 //
-Loggly.prototype.stream = function(options) {
+Loggly.prototype.stream = function(maybeOptions) {
```

Comment:
why rename to maybeOptions?

Reply:
`maybeOptions` is just a personal convention for function arguments that can be undefined. Later in the function on line 160 it assigns `options` to either the argument that gets passed, or an empty object, if it's undefined. Reassignment to the same variable is good to avoid wherever possible . The raw `maybeOptions` argument doesn't get used until you later guarantee that it contains at least an empty object, and two variables is the best way to that.